README.md

```diff
@@ -94,6 +94,20 @@ Add/edit these two attributes if not present. Set **"Status bar is initially hid
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
 
+Or in config.xml inside "platform" block:
+
+
+	<platform name="ios">
+	...
+		<config-file overwrite="true" parent="UIStatusBarHidden" platform="ios" target="*-Info.plist">
```

Comment:
Is this `platform="ios"` needed when it is already inside `<platform name="ios">`?

Reply:
```suggestion
		<config-file overwrite="true" parent="UIStatusBarHidden" target="*-Info.plist">
```

I think it can be removed, too.